testsuite/patching/src/test/java/org/jboss/as/test/patching/MergedPatchesTestCase.java

```diff
@@ -30,87 +16,75 @@
 import org.wildfly.core.testrunner.ServerControl;
 import org.wildfly.core.testrunner.WildflyTestRunner;
 
+import static org.jboss.as.patching.IoUtils.mkdir;
+import static org.jboss.as.patching.IoUtils.newFile;
+import static org.jboss.as.test.patching.PatchingTestUtil.AS_DISTRIBUTION;
+import static org.jboss.as.test.patching.PatchingTestUtil.AS_VERSION;
+import static org.jboss.as.test.patching.PatchingTestUtil.FILE_SEPARATOR;
+import static org.jboss.as.test.patching.PatchingTestUtil.MODULES_PATH;
+import static org.jboss.as.test.patching.PatchingTestUtil.PRODUCT;
+import static org.jboss.as.test.patching.PatchingTestUtil.createPatchXMLFile;
+import static org.jboss.as.test.patching.PatchingTestUtil.createZippedPatchFile;
+import static org.jboss.as.test.patching.PatchingTestUtil.randomString;
```

Comment:
The previous import ordering was correct per WildFly standards. Please adjust your IDE to either not reorder or to use the standard order. Thanks.